uint8 type
string id
uint8 TYPE_STUDENT=1
uint8 TYPE_STAFF=2
uint8 TYPE_UNKNOWN=3